webapp/src/pages/Dashboard.tsx

```diff
@@ -97,6 +99,25 @@ const Dashboard = () => {
           />
         </PreviewCard>
       )}
+      {isPipelineSelected(pipeline) && (
+        <PreviewCard
+          title="Smart Tags Aggregation"
```

Comment:
Minimally:
```suggestion
          title="Smart Tag Aggregation"
```

But I wonder if there's a better word than "Aggregation" (although I know that's what I've been calling it myself 🙃), both to be a little broader than the technical action leading to the plot, and maybe also to be consistent with the rest of the Dashboard. So far, we use "Testing" and "Analysis" on the Dashboard...I was going to suggest "Summary," but "Analysis" would also work.
```suggestion
          title="Smart Tag Analysis"
```

```suggestion
          title="Smart Tag Summary"
```

What do you think?

Reply:
I have no attachment for "aggregation". I think this plot is too verbose to be a "summary". I'll go with "analysis". Thank you! :)